src/Tester/StorageTestConstants.cs

```diff
@@ -0,0 +1,144 @@
+        public const string ORLEANS_TEST_STORAGE_KEY_FILE_NAME = "OrleansTestStorageKey.txt";
+
+        private const string DEFAULT_STORAGE_DATA_CONNECTION_STRING = "UseDevelopmentStorage=true";
+
+        public static void Init()
+        {
+            if (DataConnectionString != null)
+            {
+                return; // already initialized
+            }
+            if (File.Exists(ORLEANS_TEST_STORAGE_KEY_FILE_NAME))
+            {
+                using (TextReader input = File.OpenText(ORLEANS_TEST_STORAGE_KEY_FILE_NAME))
+                {
+                    string line = input.ReadToEnd();
+                    line = line.Trim();
+                    if (!String.IsNullOrEmpty(line))
+                    {
+                        Console.Out.WriteLine("Found the {0} file and using the Storage Key from there.", ORLEANS_TEST_STORAGE_KEY_FILE_NAME);
+                        Trace.WriteLine("Found the {0} file and using the Storage Key from there.", ORLEANS_TEST_STORAGE_KEY_FILE_NAME);
+                        DataConnectionString = line;
+                    }
+                }
+            }
+
+            if (DataConnectionString != null) return;
+
+            // If did not find the file, just use the DevelopmentStorage
+            Console.Out.WriteLine("Did not find the {0} file or it was empty. Using Default Storage Data Connection String instead.", ORLEANS_TEST_STORAGE_KEY_FILE_NAME);
+            Trace.WriteLine("Did not find the {0} file or it was empty. Using Default Storage Data Connection String instead.", ORLEANS_TEST_STORAGE_KEY_FILE_NAME);
+            DataConnectionString = DEFAULT_STORAGE_DATA_CONNECTION_STRING;
+        }
+
+        public static FileInfo GetDbFileLocation(DirectoryInfo dbDir, string dbFileName)
+        {
+            if (!dbDir.Exists) throw new FileNotFoundException("DB directory " + dbDir.FullName + " does not exist.");
+
+            string dbDirPath = dbDir.FullName;
+            var dbFile = new FileInfo(Path.Combine(dbDirPath, "TestDb.mdf"));
+            Console.WriteLine("DB file location = {0}", dbFile.FullName);
+
+            // Make sure we can write to local copy of the DB file.
+            MakeDbFileWriteable(dbFile);
+
+            return dbFile;
+        }
+
+        public static string GetSqlConnectionString(TestContext context)
+        {
+            string dbFileName = @"TestDb.mdf";
+            string dbDirPath = context.DeploymentDirectory;
+            return GetSqlConnectionString(new DirectoryInfo(dbDirPath), dbFileName);
+        }
+        public static string GetSqlConnectionString(DirectoryInfo dbDir, string dbFileName)
+        {
+            var dbFile = GetDbFileLocation(dbDir, dbFileName);
+
+            Console.WriteLine("DB directory = {0}", dbDir.FullName);
+            Console.WriteLine("DB file = {0}", dbFile.FullName);
+
+            string connectionString = string.Format(
+                @"Data Source=(LocalDB)\v11.0;"
```

Comment:
Hmm, I thought that with VS 2013 one would get SQL Server 2014 (if VS is updated). This seem to be a bit more complicated, taking a cue from EF issue [here](https://entityframework.codeplex.com/workitem/2246). I'd be inclined to go for 2014 variery as it has a flag for SQL Server Azure feature set, but let's go with v11.0 for now. This can be refactored. :)